### DISCUSSION TITLE
Where to set Growatt server IP

### DISCUSSION THREAD

**Discussion by DocZoid** [johanmeijer/grott#282] [Q&A]
Hi, sorry for the dumb question. Where would I configure my Shine WIFI to use a local proxy IP instead of server.growatt.com? The "Rerouting the network" documentation has a broken links and too few information. I could find a settings dialog in server.growatt.com -> dashboard -> settings, but I cannot set any server or IP address there.
Thanks!

**Comment by johanmeijer** [comment]
I fixed the Wiki page. 

So can set the IP address via the shinewifi (shinelan, shinelink) setup dialogue (e.g. by resetting the stick) or via the growatt site (all devices - datalogger settings menu).

**Reply by SebastianGode** [reply]
Hey @johanmeijer 
I'm getting my Growatt inverter soon.
What would happen if I change the IP on the growatt server site and then grott doesn't work? Am I unable to change it back (as all requests then fail) or is it still possible to change it back to empty?

**Reply by johanmeijer** [reply]
As descriped above. You can do this via the dialogue of the datalogger (e.g. shinelink of shinewifi)
